CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # OverReact Changelog
 
+## 2.1.0
+
+> Complete `2.1.0` Changsets:
+>
+> - [Dart 2](https://github.com/Workiva/over_react/compare/2.0.0+dart2...2.1.0)
+> - Dart 1 (no changes)
+
+- [#254][#254] Add IDE snippets (WebStorm/IntelliJ and VSCode) for Dart2-only
```

Comment:
Shouldn't need the extra `[#254]` unless we've run out of reference links at the bottom of this file.